### PR TITLE
Ops 2476

### PR DESCRIPTION
**Pull request by vikkasyousaf**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,22 @@
-# docker-python
\ No newline at end of file
+# Docker Python base image
+
+[![Build Status](https://travis-ci.org/Flaconi/docker-python.svg?branch=master)](https://travis-ci.org/Flaconi/docker-python)
```
Comment (cytopia) [inline README.md]: This is building on **travis.com**, not on **travis.org**

**Review by cytopia — CHANGES_REQUESTED** [review]
Looks good, one minor typo in the URL to adjust.
 :+1:

**Comment by cytopia** [comment]
Please also rebase your branch, otherwise you won't be able to merge it here.

**Review by vikkasyousaf — COMMENTED** [review]
Typo mistake resolved in URL

File: README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Docker Python base image
 
-[![Build Status](https://travis-ci.org/Flaconi/docker-python.svg?branch=master)](https://travis-ci.org/Flaconi/docker-python)
+[![Build Status](https://travis-ci.org/Flaconi/docker-python.svg?branch=master)](https://travis-ci.com/Flaconi/docker-python)
```
Comment (cytopia) [inline README.md]: Still using `.org` instead of `.com`

**Review by cytopia — CHANGES_REQUESTED** [review]

**Review by cytopia — APPROVED** [review]

**Pull request merged**